data_diff/dbt.py

```diff
@@ -191,7 +192,16 @@ def _get_diff_vars(
         dev_qualified_list = [dev_database, dev_schema, model.alias]
         prod_qualified_list = [prod_database, prod_schema, model.alias]
 
-    return DiffVars(dev_qualified_list, prod_qualified_list, primary_keys, dbt_parser.connection, dbt_parser.threads)
+    where = None
+    if model.meta:
+        try:
+            where = model.meta["datafold"]["datadiff"]["filter"]
```

Comment:
very open to suggestion here -- not very pretty

Reply:
**nit**: For this or future PRs.

A datafold datadiff section might contain other useful different parameters such as include/exclude_columns, time_travel columns etc which we may want to use in the future for running cloud diffs.

I think it is worth to have a special classmethod of `DbtParser` or separate function to extract from a model that section if it exists.
```
           # dbt_parser.py
           class TDatafoldDatadiffModelParams(pydantic.BasicModel):
               filter_: Optional[str] = None
               include_columns: Optional[List[str]] = None
               exclude_columns: Optional[List[str]] = None

            class DbtParser:  
                  @classmethod
                  def get_datafold_datadiff_params(cls, model) -> TDatafoldDatadiffModelParams:
                      # implementation
            
            # dbt.py     
            datafold_datadiff_params = dbt_parser.get_datafold_datadiff_params()
            return DiffVars(
                 dev_qualified_list, prod_qualified_list, primary_keys, dbt_parser.connection, dbt_parser.threads, datafold_datadiff_params.filter_)
```

Reply:
good point. I'm about to add include/exclude columns and I will refactor in that PR